VelocityPacketTrackerFirstObjective.ipynb

```diff
@@ -0,0 +1,3493 @@
+{
```

Comment:
<p>This is good but we're interested in seeing count of each species (e.g. Si II) so you need to take ion number into account too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2538/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Updated